src/hardware/serialport/serialdummy.h

```diff
@@ -26,21 +26,21 @@
 
 class CSerialDummy : public CSerial {
 public:
-	CSerialDummy(Bitu id, CommandLine* cmd);
+	CSerialDummy(const uint8_t port_idx, CommandLine *cmd);
```

Comment:
I just checked and it seems to be ok for integer parameters to have `const` omitted from declaration, but still used in definition. Please remove `const` from declaration, as it does not make a difference for person using the interface.

Reply:
Great; thanks for looking into this; will drop the `const`'s in the headers and keep them in the bodys.

I think we've found a happy balance, as I prefer to lock-down all inbound arguments to whatever they were at the time of passing,  and only permit local changes to those that we really do intend (or need to) modify inside the function.